### PR TITLE
Vs ard lora

### PR DESCRIPTION
**Pull request by Lenardgeorge**
For tutorials:
- [*] I double checked the wiring in the schematics/images
- [*] I exported the cover image without background
- [*] I used SVG format for pure vector images
- [*] I updated the metadata.json file of the tutorial
- [*] I updated the metadata.json file of the previous tutorial to contain the "next" link
- [*] I did a spell checking before requesting a review
- [*] I used title case for titles

### CODE REVIEW

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+It's now time to connect your Portenta H7 and Lora Vision Shield to TTN. You'll need to upload code to the board, so as you probably already know, there are two options:
+
+* Use the [Arduino Web Editor](https://create.arduino.cc/editor) (this is the option this guide will follow)
+* Use the [Arduino IDE](https://www.arduino.cc/en/software), this way requires manual update of Arduino core and libraries.
+
+Plug the Vision Shield to the Portenta H7 and them to your PC through the usb port. Be sure to have selected the right board "Arduino Portenta H7 (M7 core)" and the right port.
+
+![vs-ard-select-port](assets/vs_ard_select_port.png)
+
+The LoRa module on the Vision Shield can be accessed by using the [MKRWAN library](https://github.com/arduino-libraries/MKRWAN). This library provides all the APIS to communicate with LoRa and LoRaWAN networks and can be Installed from the library Manager. The first code you need to upload and run is from the **MKRWAN** library, and its name is _FirstConfiguration_.
```
Comment (GDJose) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:76]:
```suggestion
The LoRa module on the Vision Shield can be accessed by using the [MKRWAN library](https://github.com/arduino-libraries/MKRWAN)( if you can't find it in your examples list, you can go to **tools > library manager** and type "MKRWAN library" to install it). This library provides all the APIS to communicate with LoRa and LoRaWAN networks and can be Installed from the library Manager. The first code you need to upload and run is from the **MKRWAN** library, and its name is _FirstConfiguration_.
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+The only line you may need to change before uploading the code is the one that sets the frequency. Set the frequency code according to your country if needed. You can find more information about frequency by country at [this TTN link](https://www.thethingsnetwork.org/docs/lorawan/frequency-plans.html).
+
+```
+... // change this to your regional band (eg. US915, AS923, ...)
+ if (!modem.begin(EU868)) {    ...
+```
+
+[note]
+
+Please, consider that in Australia the boards connect correctly to TTN gateways on AS923 frequencies; AU915 frequencies requires the selection of sub band 2, not yet implemented in the firmware.
```
Comment (GDJose) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:89]:
```suggestion
**Note:** Consider that in Australia the boards connect correctly to TTN gateways on AS923 frequencies; AU915 frequencies requires the selection of sub band 2, not yet implemented in the firmware.
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+[/note]
+
+Once the sketch completes upload, open the Serial Monitor. The following details will show:
```
Comment (GDJose) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:93]:
```suggestion
Once you have added to the sketch the frequency according to your country, you can upload it to the board. Then, once the upload is completed open the Serial Monitor. The following details will show:
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+```
+Your module version is: ARD-078 1.1.9
```
Comment (GDJose) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:96]:
```suggestion
Your module version is: ARD-078 1.2.0
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+Your device EUI is: a8xxxxxxxxxxxx0a
```
Comment (GDJose) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:97]:
```suggestion
Your device EUI is: a8xxxxxxxxxxxxxx
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+Are you connecting via OTAA (1) or ABP (2)?
```
Comment (GDJose) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:98]: @Lenardgeorge @i-herrera should the user select 1? or 2? or do nothing?

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+
+![vs_ard_ttn_click_register](assets/vs_ard_ttn_click_register.png)
+
+On the registration page that shows up we must fill in two fields. The first one is the **Device ID** which this must be lowercase and without spaces. Since the ID should be unique, so you can use  the name of the board followed by the first and the last byte of the EUI. The second one is the **EUI** which is exactly the device EUI that appeared in the Serial Monitor. Copy paste the EUI from the Serial Monitor into the EUI field
+
+![vs_ard_ttn_register_device](assets/vs_ard_ttn_register_device.png)
+
+After pressing the Register button, your board will show up on the **Device Overview** page. You can now see all the information needed to complete the Arduino setup.
+
+![vs_ard_ttn_device_overview](assets/vs_ard_ttn_device_overview.png)
```
Comment (GDJose) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:113]: @Lenardgeorge in the screenshot the app ID and the device ID don't match with the above description on how to name the device neither the previous screenshots

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+[/note]
+
+Once the sketch completes upload, open the Serial Monitor. The following details will show:
+
+```
+Your module version is: ARD-078 1.1.9
+Your device EUI is: a8xxxxxxxxxxxx0a
+Are you connecting via OTAA (1) or ABP (2)?
+```
+
```
Comment (GDJose) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:100]:
```suggestion
[note]
**Note:** In order to select the way in which the board is going to connect with TTN  (OTAA or ABP) we need to configure it on the TTN portal. We will see which option we should select in the following steps.
[/note]
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+After pressing the Register button, your board will show up on the **Device Overview** page. You can now see all the information needed to complete the Arduino setup.
+
+![vs_ard_ttn_device_overview](assets/vs_ard_ttn_device_overview.png)
+
+### 5. Connecting to TTN
+
+Once your board has been registered you can send information to TTN. Let's come back to the Serial Monitor and proceed. It will ask for:
+
+* activation mode (that in this case is OTAA),
```
Comment (GDJose) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:119]:
```suggestion
* activation mode (that, in this case, is OTAA as you can see in the screenshot above),
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+-   Combine LoRaWAN protocol with an OpenMV example to develop your own IoT application. Take advantage of the Vision Shield's camera to detect, filter, classify images, read QR codes or more.     
+
+## Troubleshooting
+
+The most common issue is that the device cannot connect to a TTN gateway. Again, it is a good idea to check if we have coverage in the area we are conducting this tutorial, by checking out [this map](https://www.thethingsnetwork.org/map).
+
+If we are within good range of a gateway, we should also try to move our device and antenna to a window, and even hold it out the window and move it around. This has proven successful on numerous accounts, as the signal can travel less obstructed.
+
+**Authors:** Karl Söderby, Ignacio Herrera
+**Reviewed by:** Lenard George [18.03.2020]
```
Comment (GDJose) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:156]:
```suggestion
**Reviewed by:** Jose Garcia [02.02.2021]
```

**Review by GDJose — CHANGES_REQUESTED** [review]

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 11]
```diff
@@ -0,0 +1,157 @@
+# Connecting the Vision Shield to TTN using LoRa
+## Overview 
+
+This tutorial explains how to connect your Portenta H7 to The Things Network (TTN) using the the Vision Shield's Lora Connectivity feature. A data communication channel will be enabled between the H7 and a TTN application that will be configured on your TTN console.
+
+[note]
+
+**Note :** Ensure that you are within range (10 Km) from an available Lora Gateway before you try this tutorial. Check availability on [The Things Network map](https://www.thethingsnetwork.org/map).
```
Comment (lunnea) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:8]:
```suggestion
**Note :** Ensure that you are within range (10 Km) from an available LoRa Gateway before you try this tutorial. Check availability on [The Things Network map](https://www.thethingsnetwork.org/map).
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+-   [Portenta H7 board](https://store.arduino.cc/portenta-h7)
+-   [Vision Shield LoRa](https://store.arduino.cc/portenta-vision-shield-lora)
+-   [1x Dipole Pentaband antenna](https://store.arduino.cc/antenna) or a UFL Antenna
+-   Arduino IDE ([online](https://create.arduino.cc/) or [offline](https://www.arduino.cc/en/main/software))
+-   USB C cable (either USB A to USB C or USB C to USB C)
+-   An [account](https://account.thethingsnetwork.org/users/login) with The Things Network
+
+## Instructions 
+
+The Vision Shield LoRa can be connected to the TTN and can transmit data to other devices connected to this network through a secure channel. This channel is nothing but an applicaiton on the TTN network dedicated for your board. In this tutorial, you will be guided through a step-by-step process of setting up your Portenta board and the Vision Shield Lora to communicate with a TTN application. As stated before, to be able to follow this guide, to be under coverage of one of the TTN gateways. You can check for the coverage now if you have not done so yet.
```
Comment (lunnea) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:29]:
```suggestion
The Vision Shield LoRa can be connected to the TTN and can transmit data to other devices connected to this network through a secure channel. This channel is nothing but an applicaiton on the TTN network dedicated for your board. In this tutorial, you will be guided through a step-by-step process of setting up your Portenta board and the Vision Shield LoRa to communicate with a TTN application. As stated before, you need to check for the coverage of one of the TTN gateways before you continue following this guide.  
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+
+### 1.  Setting up the Environment
+
+Start by pointing your browser to www.thethingsnetwork.org and use the Sign Up button to setup an account. Next, then fill all the required fields to complete a new registration (skip this step and sign in directly if you already have a TTN account).
```
Comment (lunnea) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:33]:
```suggestion
Start by pointing your browser to www.thethingsnetwork.org and use the Sign Up button to setup an account. Next, then fill all the required fields to complete a new registration (if you already have a TTN account, skip this step and continue by signing in).
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+
+![vs_ard_things_nw](assets/vs_ard_ttn_home.png)
+
+### 2.  Creating an app on TTN
+
+Once you have created an account with TTN, you need to create a TTN [application](https://www.thethingsnetwork.org/docs/applications/). An application provides a way to aggregate data from different devices, and then use these data with other online / offline tools. Go to your [console](https://console.thethingsnetwork.org), and click on **Applications**
```
Comment (lunnea) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:39]:
```suggestion
Once you have created an account with TTN, you need to create a TTN [application](https://www.thethingsnetwork.org/docs/applications/). This(?) application provides a way to aggregate data from different devices, and then use these data with other online / offline tools. Go to your [console](https://console.thethingsnetwork.org), and click on **Applications**
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 15]
```diff
@@ -0,0 +1,157 @@
...
+
+```
+Your module version is: ARD-078 1.1.9
+Your device EUI is: a8xxxxxxxxxxxx0a
+Are you connecting via OTAA (1) or ABP (2)?
+```
+
+### 4. Registring the Portenta on TTN
+
+Before your Portenta H7 can start communicating with the TTN you need to [register](https://www.thethingsnetwork.org/docs/devices/registration.html) it with an application. Go back to the TTN portal and scroll to the **Devices** section on your Application dashboard, then click on  **Register Device** button.
```
Comment (lunnea) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:103]:
```suggestion
Before your Portenta H7 can start communicating with the TTN you need to [register](https://www.thethingsnetwork.org/docs/devices/registration.html) the board with an application. Go back to the TTN portal and scroll to **Devices** section on your Application dashboard, then click **Register Device**.
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+
+![vs_ard_ttn_click_register](assets/vs_ard_ttn_click_register.png)
+
+On the registration page that shows up we must fill in two fields. The first one is the **Device ID** which this must be lowercase and without spaces. Since the ID should be unique, so you can use  the name of the board followed by the first and the last byte of the EUI. The second one is the **EUI** which is exactly the device EUI that appeared in the Serial Monitor. Copy paste the EUI from the Serial Monitor into the EUI field
```
Comment (lunnea) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:107]:
```suggestion
On the registration page, fill in **Device ID** and **EUI**. 
**Note**:  The Device ID must be lowercase and without spaces. The **EUI** should be copied from the Serial Monitor.```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+
+```
+Your module version is: ARD-078 1.1.9
+Your device EUI is: a8xxxxxxxxxxxx0a
+Are you connecting via OTAA (1) or ABP (2)?
+Enter your APP EUI
+Enter your APP KEY
+```
+
+Next, introduce the **APP EUI** and the **APP KEY** in the Serial Monitor in a similar manner. If everything has gone fine, you will see this message:
```
Comment (lunnea) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:133]:
```suggestion
Next, introduce the **APP EUI** and the **APP KEY** in the Serial Monitor. If this process is done successfully, you will see this message:
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 18]
```diff
@@ -0,0 +1,157 @@
...
+
+```
+Message sent correctly!
+```
+
+## Conclusion
+
+If you have sent the message successfuly you have configured Portenta H7 and the Lora Vision Shield on the TTN. We have retrieved the device EUI, used it to register the device in the TTN console, and programmed the board using the data provided by TTN. We can now send data over the LoRa® network which can be viewed from anywhere in the world (as long as we have an Internet connection and our device is in range from a TTN gateway).
```
Comment (lunnea) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:141]:
```suggestion
If you recieve this message, you have managed to configure the Portenta H7 and the Lora Vision Shield on the TTN.
 We have retrieved the device EUI, used it to register the device in the TTN console, and programmed the board using the data provided by TTN. Now, we can send data over the LoRa® network which can be viewed from anywhere in the world (as long as we have an Internet connection and our device is in range from a TTN gateway).
```

File: content/tutorials/portenta-h7/vs-ard-lora/content.md [hunk 19]
```diff
@@ -0,0 +1,157 @@
...
+
+### Next Steps
+
+-   Try sending uplink and downlink messages between Portenta and your TTN application with _LoraSendAndReceive_ sketch from the MKRWAN library.
+-   Experiment your Vision Shield's capabilities with OpenMV and the examples from the dedicated library for Arduino. You can continue with [this tutorial](https://www.arduino.cc/pro/tutorials/portenta-h7/por-openmv-bt) from the Arduino Pro site.
+-   Combine LoRaWAN protocol with an OpenMV example to develop your own IoT application. Take advantage of the Vision Shield's camera to detect, filter, classify images, read QR codes or more.     
+
+## Troubleshooting
```
Comment (lunnea) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:149]: General thought: Is it possible to provide different use cases in the Troubleshooting sections that entails "normal" issues that users may encounter?

File: content/tutorials/portenta-h7/vs-ard-lora/content.md (hunk 10 above)
Comment (lunnea) [inline content/tutorials/portenta-h7/vs-ard-lora/content.md:156]:
```suggestion
**Reviewed by:** Linnea Åkerberg [03.02.2021] + 

```

**Review by lunnea — COMMENTED** [review]
Proofreading 3/2 2021

**Review by Lenardgeorge — COMMENTED** [review]
All changes have been reviewed

**Review by Lenardgeorge — COMMENTED** [review]
All changes have been reviewed

**Pull request merged**